.github/test_result_consistency.py

```diff
@@ -136,23 +161,47 @@ def coord_test(
         ):
             print(f"- allow missing coordinate: {expected_coord_name} in variable {data_var_name}")
             continue
+
+        multiplier = 1
+        if expected_coord_name in ["rate", "lifetime"]:
+            multiplier = -1
```

Comment:
@jsnel This is the part in the tests where we account for the changed sign of `rate` and `lifetime` (negative -> positive)